browser/cypress.config.ts

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
I'm surprised that TS thinks `defineConfig` is "extraneous", seeing as we use it right below